### ISSUE TITLE
[Feature Request] Compare weights before and after

### ISSUE THREAD

**Issue by cjdduarte** [open-spaced-repetition/fsrs4anki#162]
**Which module is related to your feature request?**
Simulator

**Additional context**

In the last optimization my weights were:
var w = [0.9342, 0.9359, 5.0775, -0.5318, -0.6451, 0.0895, 1.3573, -0.1117, 0.7573, 1.9057, -0.2947, 0.1673, 0.93];

today, I ran the optimizer again and it gave me these new weights:
var w = [0.9323, 0.933, 5.0763, -0.5305, -0.6284, 0.1012, 1.3545, -0.1119, 0.7544, 1.903, -0.2977, 0.1652, 0.9221];

I would like to understand why the new recommendation.
I am "better" or "worse" as time goes by.
Do I retain better at first or better with time?

I thought of using the simulator to compare the two weights: before and after, along with the Anki standard. It's possible?
**There would appear 3 lines: Anki standard, old weight and new weight**

![image](https://user-images.githubusercontent.com/42243532/220730751-57234fce-a1c7-4474-9ef8-d7615948a999.png)

**Another topic:** 
Does this graph make sense as output from the simulator?
![image](https://user-images.githubusercontent.com/42243532/220721226-2d12f22b-3840-4106-ace5-7279f6f42cda.png)

**Comment by L-M-Sherlock** [comment]
> Does this graph make sense as output from the simulator?

This data generated by the optimizer could be used to draw the graph:

![image](https://user-images.githubusercontent.com/32575846/221073632-a953ffed-9afc-4abc-b986-1179b08559fd.png)

**Comment by L-M-Sherlock** [comment]
> I would like to understand why the new recommendation.
> I am "better" or "worse" as time goes by.
> Do I retain better at first or better with time?

The metric to compare the parameters of the model is the loss:

![image](https://user-images.githubusercontent.com/32575846/221073922-fb6b5774-7b82-499c-aff0-cf6f15c84b36.png)

**Comment by cjdduarte** [comment]
What is the unit of measure for this loss?
how do i read this?

Loss before training: 0.4109
Loss after training: 0.4053

**Comment by L-M-Sherlock** [comment]
> Loss before training: 0.4109
> Loss after training: 0.4053

The less, the better. The loss is BCLoss, aka Binary Cross Entropy.

**Comment by cjdduarte** [comment]
Tanks
